notebooks/official/explainable_ai/sdk_automl_tabular_binary_classification_batch_explain.ipynb

```diff
@@ -856,7 +856,7 @@
         "    gcs_source=gcs_input_uri,\n",
```

Comment:
<p>remove future tense -&gt; will</p><p> </p><p>remove: - we just want to demonstrate how to make a prediction.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/829/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>